src/core/ext/filters/http/message_compress/message_compress_filter.cc

```diff
@@ -77,13 +77,27 @@ class ChannelData {
               name);
       default_compression_algorithm_ = GRPC_COMPRESS_NONE;
     }
+
+    default_grpc_min_message_size_to_compress_ = grpc_core::DefaultGrpcMinMessageSizeToCompressFromChannelArgs(
+      args->channel_args
+    );
+
+    compression_options_ = grpc_core::MakeCompressionOptions(args->channel_args);
     GPR_ASSERT(!args->is_last);
   }
 
   grpc_compression_algorithm default_compression_algorithm() const {
     return default_compression_algorithm_;
   }
 
+  const grpc_core::CompressionOptions* default_compression_options() {
+    return compression_options_.get();
+  }
+
+  int default_grpc_min_message_size_to_compress() const {
```

Comment:
Please call this `min_message_size_to_compress()`.

Reply:
done